common/src/main/java/org/apache/rocketmq/common/protocol/header/CheckTransactionStateRequestHeader.java

```diff
@@ -20,18 +20,19 @@
  */
 package org.apache.rocketmq.common.protocol.header;
 
-import org.apache.rocketmq.remoting.CommandCustomHeader;
+import org.apache.rocketmq.common.rpc.RpcRequestHeader;
 import org.apache.rocketmq.remoting.annotation.CFNotNull;
 import org.apache.rocketmq.remoting.exception.RemotingCommandException;
 
-public class CheckTransactionStateRequestHeader implements CommandCustomHeader {
+public class CheckTransactionStateRequestHeader extends RpcRequestHeader {
     @CFNotNull
     private Long tranStateTableOffset;
     @CFNotNull
     private Long commitLogOffset;
     private String msgId;
     private String transactionId;
     private String offsetMsgId;
+    private int queueId;
 
```

Comment:
The CheckTransactionStateRequestHeader is sent by broker. I think we should change the org.apache.rocketmq.broker.client.net.Broker2Client#checkProducerTransactionState to set the new fileds

Reply:
It's only client-side modification.